package.json

```diff
@@ -22,32 +22,38 @@
     "src/components"
   ],
   "dependencies": {
-    "babel-preset-react-app": "^3.1.1",
+    "@types/classnames": "^2.2.4",
+    "@types/js-yaml": "^3.11.1",
+    "@types/react": "^16.3.17",
+    "@types/react-dom": "^16.0.6",
+    "@types/react-helmet": "^5.0.6",
     "classnames": "^2.2.6",
-    "css-loader": "0.28.7",
-    "file-loader": "1.1.5",
     "gitdiff-parser": "^0.0.8",
     "js-yaml": "^3.11.0",
     "object-assign": "^4.1.1",
     "promise": "^8.0.1",
     "react": "^16.3.2",
     "react-dom": "^16.3.2",
     "react-helmet": "^5.2.0",
-    "style-loader": "0.19.0",
-    "url-loader": "0.6.2",
-    "webpack": "3.8.1",
-    "webpack-dev-server": "2.9.4",
+    "styled-components": "^3.3.2",
     "whatwg-fetch": "2.0.3"
   },
   "scripts": {
     "start": "node scripts/start.js",
     "test": "echo Test Passed!"
   },
   "devDependencies": {
-    "babel-loader": "^7.1.4",
-    "babel-plugin-react-require": "^3.0.0",
+    "awesome-typescript-loader": "^5.0.0",
     "copy-webpack-plugin": "^4.5.1",
+    "css-loader": "^0.28.11",
+    "file-loader": "^1.1.11",
     "filewatcher-webpack-plugin": "^1.2.0",
-    "html-webpack-plugin": "^3.2.0"
+    "html-webpack-plugin": "^3.2.0",
+    "style-loader": "^0.21.0",
+    "tslint": "^5.10.0",
```

Comment:
Since you are adding TSLint, please add **lint** script in package.json and Travis CI (you can refer to [tuture-cli](https://github.com/tutureproject/tuture) repo for how to do this).

Reply:
I need sometime to customize my TSLint rules, So I decide to add it in next PR.